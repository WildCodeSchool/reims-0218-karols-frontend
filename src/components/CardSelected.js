import React from "react"
import {
  Card,
  CardTitle,
  CardText,
  CardImg,
  CardImgOverlay,
  CardFooter,
  Button
} from "reactstrap"

const CardSelected = ({ name, description, image }) => {
  return (
    <div>
      <Card inverse>
        <CardImg width="100%" src={image} />
        <CardImgOverlay>
          <CardTitle
            style={{
              fontSize: "2em"
            }}
          >
            {name}
          </CardTitle>
          <CardText
            style={{
              fontSize: "1.5em"
            }}
          >
            {description}
          </CardText>
        </CardImgOverlay>
        <CardFooter>
          <Button color="primary" size="lg" block>
            Choisir
          </Button>
        </CardFooter>
      </Card>
    </div>
  )
}

export default CardSelected
