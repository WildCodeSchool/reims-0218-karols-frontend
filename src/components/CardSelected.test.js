import React from "react"
import ReactDOM from "react-dom"
import CardSelected from "./CardSelected"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<CardSelected />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<CardSelected />).toJSON()
  expect(tree).toMatchSnapshot()
})
