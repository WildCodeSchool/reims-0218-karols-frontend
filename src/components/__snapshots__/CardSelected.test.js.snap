// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="card text-white"
  >
    <img
      className="card-img"
      src={undefined}
      width="100%"
    />
    <div
      className="card-img-overlay"
    >
      <h5
        className="card-title"
        style={
          Object {
            "fontSize": "2em",
          }
        }
      />
      <p
        className="card-text"
        style={
          Object {
            "fontSize": "1.5em",
          }
        }
      />
    </div>
    <div
      className="card-footer"
    >
      <button
        className="btn btn-primary btn-lg btn-block"
        onClick={[Function]}
        type={undefined}
      >
        Choisir
      </button>
    </div>
  </div>
</div>
`;
